analytics-webapps/src/main/webapp/vue-app/common-components/components/samples/SampleItemAttribute.vue

```diff
@@ -0,0 +1,151 @@
+<!--
+  This file is part of the Meeds project (https://meeds.io/).
+  Copyright (C) 2022 Meeds Association
+  contact@meeds.io
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 3 of the License, or (at your option) any later version.
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public License
+  along with this program; if not, write to the Free Software Foundation,
+  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+-->
+<template>
+  <v-row>
+    <v-col>{{ getI18N(value) }}</v-col>
+    <v-col v-if="sampleItemExtension && initialized">
+      <extension-registry-component
+        :component="extendedSampleItemComponent"
+        :params="extendedSampleItemComponentParams" />
+    </v-col>
+    <v-col v-else-if="value === 'userId' && userIdentity" class="text--secondary">
+      {{ chartData[value] }}
+      (<analytics-profile-chip :identity="userIdentity" />)
+    </v-col>
+    <v-col v-else-if="value === 'spaceId' && spaceIdentity" class="text--secondary">
+      {{ chartData[value] }}
+      (<analytics-profile-chip :identity="spaceIdentity" />)
+    </v-col>
+    <v-col v-else-if="value === 'modifierSocialId' && userModifierIdentity" class="text--secondary">
+      {{ chartData[value] }}
+      (<analytics-profile-chip :identity="userModifierIdentity" />)
+    </v-col>
```

Comment:
```suggestion
```

It would be better to define those three elements as extensions as well IMO to demonstrate how it's done (it will serve as a sample for other developers)